azure/deploy-to-azure/azure-integration/CreateUIDefinition-managementgroups.json

```diff
@@ -335,7 +346,8 @@
       "automute": "[steps('datadogConfig').metrics.automute]",
       "custom_metrics_enabled": "[steps('datadogConfig').metrics.custom_metrics_enabled]",
       "resource_collection_enabled": "[steps('datadogConfig').resourceCollection.resource_collection_enabled]",
-      "cspm_enabled": "[and(steps('datadogConfig').resourceCollection.cspm_enabled, steps('datadogConfig').resourceCollection.resource_collection_enabled)]"
+      "cspm_enabled": "[and(steps('datadogConfig').resourceCollection.cspm_enabled, steps('datadogConfig').resourceCollection.resource_collection_enabled)]",
+      "excluded_resource_providers": "[steps('datadogConfig').metrics.excluded_resource_providers]"
     }
   }
```

Comment:
nit: can we ensure the newline at the end of the file isnt removed (both this and the next file)

Reply:
done